src/Misc/expressionFunc/hashFiles/src/hashFiles.ts

```diff
@@ -18,38 +13,9 @@ async function run(): Promise<void> {
   }
 
   console.log(`Match Pattern: ${matchPatterns}`)
-  let hasMatch = false
-  const githubWorkspace = process.cwd()
-  const result = crypto.createHash('sha256')
-  let count = 0
-  const globber = await glob.create(matchPatterns, {followSymbolicLinks})
-  for await (const file of globber.globGenerator()) {
-    console.log(file)
-    if (!file.startsWith(`${githubWorkspace}${path.sep}`)) {
-      console.log(`Ignore '${file}' since it is not under GITHUB_WORKSPACE.`)
-      continue
-    }
-    if (fs.statSync(file).isDirectory()) {
-      console.log(`Skip directory '${file}'.`)
-      continue
-    }
-    const hash = crypto.createHash('sha256')
-    const pipeline = util.promisify(stream.pipeline)
-    await pipeline(fs.createReadStream(file), hash)
-    result.write(hash.digest())
-    count++
-    if (!hasMatch) {
-      hasMatch = true
-    }
-  }
-  result.end()
 
-  if (hasMatch) {
-    console.log(`Found ${count} files to hash.`)
-    console.error(`__OUTPUT__${result.digest('hex')}__OUTPUT__`)
-  } else {
-    console.error(`__OUTPUT____OUTPUT__`)
-  }
+  const result = await glob.hashFiles(matchPatterns, {followSymbolicLinks})
```

Comment:
We lose some logging by doing this, can we submit a pr to the toolkit to add a verbose mode to `glob.hashfiles`. Happy to walk through that as needed!

Reply:
We can use `core.info()` instead of `core.debug()` in toolkit implementation, if that won't cause a problem in other places where the `hashFiles()` is used.